src/components/IOUConfirmationList.js

```diff
@@ -232,6 +274,32 @@ class IOUConfirmationList extends Component {
         return iouAmount !== sumAmount ? (amountPerPerson + difference) : amountPerPerson;
     }
 
+    /**
+    * Toggle selected option's selected prop.
+    * @param {Object} option
+    */
+    toggleOption(option) {
+        const isSelf = !_.some(this.state.participants, selectedOption => (
+            selectedOption.login === option.login
+        ));
+
+        if (isSelf) {
+            return;
+        }
```

Comment:
I think this would be more readable if you put the negation inside the callback:
```
const isSelf = _.some(this.state.participants, selectedOption => (
    selectedOption.login !== option.login
));
```

Reply:
yes!

Reply:
Actually, that doesn't work. `isSelf` is always true.

What I'm doing here is checking if the option is in the participants list.

So I'll use `_.every` instead
```
const isSelf = _.every(this.state.participants, selectedOption => (
    selectedOption.login !== option.login
));
```